packages/stack/proxy/src/index.ts

```diff
@@ -92,10 +92,15 @@ export default class ProxyManager {
       return;
     }
     this.inited = true;
-
-    let port = this.embark.config.blockchainConfig.rpcPort;
-    if (!port && port !== 0) {
-      port = 8545;
+    const rpcConfigPort = this.embark.config.blockchainConfig.rpcPort;
+    let port: number = 0;
```

Comment:
Just a style thingy but the type annotation here can be omitted because TS is going to infer it anyways.